internal/bminventory/inventory.go

```diff
@@ -238,7 +238,7 @@ func (b *bareMetalInventory) getUserSshKey(params installer.GenerateClusterISOPa
 		return ""
 	}
 	return fmt.Sprintf(`{
-		"name": "core",
+		"name": "systemUser",
```

Comment:
no. it should be core.

Reply:
core already exists. Not sure how to update an existing user.

Reply:
mmm worth checking. systemUser is missleading, that's why we aligned to core all over